new-integration-tests/src/test/java/oracle/weblogic/kubernetes/actions/impl/TraefikParams.java

```diff
@@ -4,25 +4,24 @@
 package oracle.weblogic.kubernetes.actions.impl;
 
 import java.util.HashMap;
+import java.util.Objects;
 
 import oracle.weblogic.kubernetes.actions.impl.primitive.HelmParams;
 
 // All parameters needed to install Traefik Operator
 
 public class TraefikParams {
 
+  // only add the values which need to be updated here
+  // the default traefik values are here: https://github.com/helm/charts/blob/master/stable/traefik/values.yaml
+  private static final String NAMESPACES = "kubernetes.namespaces";
+
   // Adding some of the most commonly used params for now
-  private int nodePortsHttp;
-  private int nodePortsHttps;
+  private String nameSpaces;
   private HelmParams helmParams;
 
-  public TraefikParams nodePortsHttp(int nodePortsHttp) {
-    this.nodePortsHttp = nodePortsHttp;
-    return this;
-  }
-
-  public TraefikParams nodePortsHttps(int nodePortsHttps) {
-    this.nodePortsHttps = nodePortsHttps;
+  public TraefikParams nameSpaces(String nameSpaces) {
+    this.nameSpaces = nameSpaces;
```

Comment:
don't use capital `S` - namespace is one word

Reply:
done